test/bdd/fixtures/universal-registrar/docker-compose.yml

```diff
@@ -4,28 +4,25 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
-version: "3.5"
-networks:
-  default:
-    name: universal-registrar
+version: '3'
+
 services:
-  driver-did-sov:
```

Comment:
how do we handle sov?

Reply:
there is issue in sov driver with local uni-registrar

Reply:
@sudeshrshetty ^^^^

Reply:
@troyronda @sudeshrshetty https://github.com/trustbloc/edge-service/pull/207/files#diff-a2c2abd8f0260e030bfc497f1d7f166fL23

Reply:
@troyronda @sudeshrshetty we handle sov using https://uniregistrar.io/ not local